src/typescript.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Can you add a fileoverview comment explaining what this does (i.e. uses 2.4 types on 2.5 API)?

Reply:
done